src/NLog/Targets/FileTarget.cs

```diff
@@ -957,15 +957,15 @@ protected override void CloseTarget()
         /// <summary>
         /// Can be used if <see cref="Target.OptimizeBufferReuse"/> has been enabled.
         /// </summary>
-        private readonly ReusableStreamCreator reusableFileWriteStream = new ReusableStreamCreator();
+        private readonly ReusableStreamCreator reusableFileWriteStream = new ReusableStreamCreator(1024);
```

Comment:
No it was empty MemoryStream-object, but since it is enabled by default, then I wanted a good default value.